### PR TITLE
Update package after RMT API change.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: src/one_wire.toit [hunk 1]
```diff
@@ -85,26 +78,46 @@ class Protocol:
...
-  static encode_write_signals_ signals/rmt.Signals bits/int --from/int=0 --count/int=8 -> none:
+  /**
+  Encodes the given integer or byte array to a sequence of signals.
+
+  The $bits_or_bytes must be either an integer, in which case the $count must be given.
+  If $bits_or_bytes is a byte array, then the $count must be equal to $bits_or_bytes * 8.
+  */
+  static encode_write_signals_ bits_or_bytes/any --count/int -> rmt.Signals:
+    // Add one bit for the termination signal.
+    signals :=  rmt.Signals (count * SIGNALS_PER_BIT)
```
Comment (kasperl) [inline src/one_wire.toit:107]: Drop one space after :=.

File: src/one_wire.toit [hunk 2]
```diff
@@ -85,26 +78,46 @@ class Protocol:
...
 
-  static encode_write_signals_ signals/rmt.Signals bits/int --from/int=0 --count/int=8 -> none:
+  /**
+  Encodes the given integer or byte array to a sequence of signals.
+
+  The $bits_or_bytes must be either an integer, in which case the $count must be given.
+  If $bits_or_bytes is a byte array, then the $count must be equal to $bits_or_bytes * 8.
+  */
+  static encode_write_signals_ bits_or_bytes/any --count/int -> rmt.Signals:
+    // Add one bit for the termination signal.
```
Comment (kasperl) [inline src/one_wire.toit:106]: Where is this added?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/one_wire.toit (hunk 1 above)
Comment (floitsch) [reply src/one_wire.toit]: done.

File: src/one_wire.toit (hunk 2 above)
Comment (floitsch) [reply src/one_wire.toit]: oops.
Now done by the RMT library and not here anymore.

**Pull request merged**